app/src/main/java/com/lagradost/cloudstream3/MainAPI.kt

```diff
@@ -1452,6 +1452,7 @@ fun TvType?.isEpisodeBased(): Boolean {
 data class NextAiring(
     val episode: Int,
     val unixTime: Long,
+    val season: Int? = null,
```

Comment:
This needs a backwards compatible constructor that holds episode and unixtime

Reply:
having the last argument as null might look like that, but is in fact not from the JVM, so it will give a fatal crash for all extentions using it

Reply:
Oh, didn't know that
Now I got why you are so careful when touching MainApi
I will do it today